nmkapp/forms.py

```diff
@@ -81,6 +81,18 @@ def clean(self):
         return cleaned_data
 
 
+class RequestDisplayNameForm(forms.Form):
+    display_name = forms.CharField(label=_('Choose Display Name'), required=True, max_length=28, min_length=3)
```

Comment:
Koje su šanse da prepopulateujemo ovo polje sa onim što vrati google, da imamo "sensible default"?:)